domains/nico.json

```diff
@@ -0,0 +1,15 @@
+{
+  "description": "Personal Website",
+  "repo": "https://github.com/NicoGaming77/NicoGaming77.github.io",
+  "owner": {
+    "username": "NicoGaming77",
+    "email": "nicodreyer7@icloud.com",
+    "discord": "nicosnico#7674"
+  }
+}
+
+{
+  "record": {
```

Comment:
Invalid JSON, the record object should come after owner, not in a seperate thing (forgot what it's called)